### PR TITLE
Added build_script and build_info for node-sass:v8.0.0

### PR DESCRIPTION
**Pull request by shubham-garud**
## Checklist

- [ ] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [ ] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by seth-priya** [comment]
@nikhil-kalbande please review

**Comment by nikhil-kalbande** [comment]
@shubham-garud  we don't need `latest` folder.  Keep all files onel in `package name` folder only.

**Comment by shubham-garud** [comment]
@nikhil-kalbande Please review this

**Comment by shubham-garud** [comment]
@nikhil-kalbande  please review this

File: n/node-sass/build_info.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "package_name":"node-sass",
+    "github_url":"https://github.com/sass/node-sass",
+    "package_dir":"n/node-sass/",
+    "version":"v8.0.0",
+    "docker_build":false,
```
Comment (nikhil-kalbande) [inline n/node-sass/build_info.json:6]: @shubham-garud  Can you please add default_branch and build_script entries in default section here.
```
    "package_name":"node-sass",
    "github_url":"https://github.com/sass/node-sass",
    "package_dir":"n/node-sass/",
    "version":"v8.0.0",
    "docker_build":false,
    "build_script":node-sass_v8.0.0_ubi_8.7.sh,
    "default_branch":master,
```

File: n/node-sass/node-sass_v8.0.0_ubi_8.7.sh [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+git checkout $PACKAGE_VERSION
+
+if ! npm install; then
+    echo "------------------$PACKAGE_NAME:install_fails-------------------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | GitHub | Fail |  Install_Fails"
+    exit 1
+fi
+
+node scripts/build -f
```
Comment (nikhil-kalbande) [inline n/node-sass/node-sass_v8.0.0_ubi_8.7.sh:38]: @shubham-garud What this command will do? Can you please explain. Is it part of build?

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

File: n/node-sass/node-sass_v8.0.0_ubi_8.7.sh (hunk 2 above)
Comment (shubham-garud) [reply n/node-sass/node-sass_v8.0.0_ubi_8.7.sh:38]: Yes it is part of build. It is also present in previous script.

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**